DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -54,8 +54,8 @@ public async Task InitializeCacheAsync()
     /// <param name="end">When the event ends. Required for <see cref="ScheduledGuildEventType.External"/></param>
     /// <param name="location">Where this location takes place.</param>
     /// <returns>The created event.</returns>
-    public async Task<DiscordScheduledGuildEvent> CreateScheduledGuildEventAsync(ulong guildId, string name, string description, ulong? channelId, ScheduledGuildEventType type, ScheduledGuildEventPrivacyLevel privacyLevel, DateTimeOffset start, DateTimeOffset? end, string location = null)
-        => await this.ApiClient.CreateScheduledGuildEventAsync(guildId, name, description, start, type, privacyLevel, new DiscordScheduledGuildEventMetadata(location), end, channelId);
+    public async Task<DiscordScheduledGuildEvent> CreateScheduledGuildEventAsync(ulong guildId, string name, string description, ulong? channelId, ScheduledGuildEventType type, ScheduledGuildEventPrivacyLevel privacyLevel, DateTimeOffset start, DateTimeOffset? end, Stream? image = null, string location = null)
```

Comment:
where's param docs for Stream :german:

Reply:
DiscordRestClient can die a thousand deaths

Reply:
you will die a thousand and one deaths if you don't fix that /j